recipe/meta.yaml

```diff
@@ -12,19 +12,24 @@ source:
   sha256: {{ sha256hash }}
 
 build:
-  number: 1
+  number: 1001
 
 requirements:
   build:
     - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
   host:
     - python
-    - setuptools
+    - pip
     - cffi >=1.8
   run:
     - python
     - cffi >=1.8
```

Comment:
I'd also remove cffi as a runtime dependency. Most users don't need this (the c extension is used by default), and the ones that do can install it as an optional dependency.